### PR TITLE
chore(deps): refresh deps to the latest

### PR DESCRIPTION
**Pull request by chenrui333**
relates to #277

### CODE REVIEW

**Comment by pitbulk** [comment]
CI is failing: https://travis-ci.org/github/onelogin/java-saml/jobs/730211399

**Comment by chenrui333** [comment]
let me check. :)

**Comment by chenrui333** [comment]
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for OneLogin java-saml Toolkit Project 2.5.1-SNAPSHOT:
[INFO]
[INFO] OneLogin java-saml Toolkit Project ................. SUCCESS [  0.427 s]
[INFO] OneLogin java-saml Toolkit Core .................... SUCCESS [  6.898 s]
[INFO] OneLogin java-saml Toolkit ......................... SUCCESS [  2.899 s]
[INFO] OneLogin java-saml Toolkit Samples ................. SUCCESS [  0.002 s]
[INFO] OneLogin java-saml Toolkit Sample Webapp ........... SUCCESS [  0.078 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  10.416 s
[INFO] Finished at: 2020-09-26T11:09:19-04:00
[INFO] ------------------------------------------------------------------------
```

The build is all green now, cc @pitbulk  :)

File: core/src/test/java/com/onelogin/saml2/test/logout/LogoutRequestTest.java [hunk 1]
```diff
@@ -735,18 +735,18 @@ public void testIsInValidSign() throws Exception {
 		assertTrue(logoutRequest.isValid());
 
 		settings.setStrict(false);
-		String signature2 = "vfWbbc47PkP3ejx4bjKsRX7lo9Ml1WRoE5J5owF/0mnyKHfSY6XbhO1wwjBV5vWdrUVX+xp6slHyAf4YoAsXFS0qhan6txDiZY4Oec6yE+l10iZbzvie06I4GPak4QrQ4gAyXOSzwCrRmJu4gnpeUxZ6IqKtdrKfAYRAcVf3333=";
+		String signature2 = "x3Yq1dQ0S/6iirAPpkEYrDvY5mTqzQ3b1eE+sEmnmYbzDs5YHksRrc7uloHt7xqBcCGlk+ZI2USjKshf//OVRkSr8gZ8qYtth1v69hVpEvUdzhSANyJCOCENN2DhX8kc76Wg+VyR1mzbvbrap0G6lrj9TSuM4wyh68gzJDeTQbs=";
```
Comment (pitbulk) [inline core/src/test/java/com/onelogin/saml2/test/logout/LogoutRequestTest.java:738]: Why an update on the dependencies alter the result of a Signature?

Comment (chenrui333) [reply core/src/test/java/com/onelogin/saml2/test/logout/LogoutRequestTest.java:738]: The old signature2 failed with the validation test (`a valid base 64 alphabet but not a possible value. Expected the discarded bits to be zero.`), that is why I updated it. 

```
testIsInValidSign(com.onelogin.saml2.test.logout.LogoutResponseTest): expected:<[Signature validation failed. Logout Response rejected]> but was:<[Last encoded character (before the paddings if any) is a valid base 64 alphabet but not a possible value. Expected the discarded bits to be zero.]>
```


Maybe the dependency upgrade started to capture it?

**Comment by pitbulk** [comment]
It seems that the update of commons-codec is the one causing the issue with the Signatures.

File: core/src/test/java/com/onelogin/saml2/test/logout/LogoutRequestTest.java (hunk 1 above)
Comment (pitbulk) [reply core/src/test/java/com/onelogin/saml2/test/logout/LogoutRequestTest.java:738]: I verified that the new version of commons-codec 1.15 fixes the issue

**Comment by pitbulk** [comment]
With commons-codec 1.15 the test without the Signature change past:
https://travis-ci.org/github/onelogin/java-saml/builds/731236854

**Pull request merged**